target/linux/ramips/image/mt76x8.mk

```diff
@@ -666,6 +666,16 @@ define Device/widora_neo-32m
 endef
 TARGET_DEVICES += widora_neo-32m
 
+define Device/hak5_wifi-pineapple-mk7
+  BLOCKSIZE := 64k
+  IMAGE_SIZE := 32448k
+  DEVICE_VENDOR := Hak5
+  DEVICE_MODEL := WiFi Pineapple Mark 7
```

Comment:
Mk7 should be DEVICE_VARIANT

Reply:
@adschm why do you think `DEVICE_VARIANT` should be used here? All other Pineapples are based on different platforms (AR9331, AR9344, IPQ40xx). And they are basically different models/products, sharing only the family name. IMHO, that would be misuse of this feature here.

Reply:
To me, Mark X sounds like a revision, essentially the same as vX for TP-Link devices etc. (which might have different platforms and targets as well, e.g. TL-WR841 ...). In my understanding it is part of the idea of DEVICE_MODEL/DEVICE_VARIANT to maintain variants/revisions across targets.

However, whether DEVICE_VARIANT is used for a "revision" or not, and what can actually be considered a revision is essentially philosophic and matter of taste eventually.
I have no specific knowledge of the devices in question, so if you come to a different conclusion I'm perfectly fine.